test/test_tseries.jl

```diff
@@ -338,15 +342,15 @@ end
     @test_throws MethodError tq + 5
     @test 5 .+ tq == tq .+ 5  # broadcasting works
     @test_throws ArgumentError tq + 5tm   # different frequencies not allowed
-    
+
     # shape errors
     @test_throws ArgumentError TimeSeriesEcon.shape_error(typeof(1), typeof(2))
     @test_throws ArgumentError TimeSeriesEcon.shape_error(1, 2)
 
     # maximum and minimum
     @test minimum(tq) == minimum(tq.values)
     @test maximum(tq) == maximum(tq.values)
-    halve(x) = x/2
+    halve(x) = x / 2
```

Comment:
Spaces gets removed around `+` signs but added around `/` signs?

Reply:
I just hit Alt+Shift+F and let the Julia extension in VSCode worry about formatting so I don't have to :-)

I think indexing expressions (inside `[ ]`) have no spaces around arithmetic operators, whereas in other expressions there are.